pkg/handler3/handler.go

```diff
@@ -193,6 +226,8 @@ func (o *OpenAPIService) HandleGroupVersion(w http.ResponseWriter, r *http.Reque
 				return
 			}
 			w.Header().Set("Etag", etag)
+
+			ApplyCacheBusting(w, r, etag)
 			http.ServeContent(w, r, "", lastModified, bytes.NewReader(data))
```

Comment:
In the case that `ApplyCacheBusting` calls `http.Redirect`, does this `http.ServeContent` conflict with the response?